src/Pelagos/Bundle/AppBundle/Resources/public/js/sidebyside.js

```diff
@@ -96,6 +106,131 @@ $(document).ready(function()
         $("#right").load(getFormUrl + "/" + udi + "/" + version, function() {
             $(".smallmap", this).gMap();
             $(".filetabs", this).tabs();
+            rightLoaded = true;
+            loading.notify();
         });
     });
-});
\ No newline at end of file
+
+    $("#show-diff-button").button("option", "disabled", true).click(function (){
+        showDifferences();
+    });
+
+    loading.progress(function() {
+        if (leftLoaded && rightLoaded) {
+            $("#show-diff-button").button("enable");
+        } else {
+            $("#show-diff-button").button("disable");
+        }
+    });
+
+    function showDifferences()
+    {
+        // Change textarea and input to divs that look like them.
+        $("#right")
+        .find("textarea,input[type=text]")
+        .not(".keywordinput")
+        .each(function() {
+            var originalInput = $(this);
+            var inputType = originalInput.prop("tagName");
```

Comment:
Used only once, not required to create a variable for this. Similarly for inputWidth ,inputHeight, inputText, attrs

Reply:
nested